src/main/secure-local-store.ts

```
@@ -0,0 +1,49 @@
+import path from "node:path";
+import fs from "node:fs";
+import { app, safeStorage } from "electron";
+
+export class SecureLocalStore {
+  private path: string;
+
+  constructor(filename: string) {
+    this.path = path.join(app.getPath("userData"), `${filename}.encrypted`);
+  }
+
+  get(key: string): any {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)